### PR TITLE
[pvr] refactoring - review fixup after merge of 16677

### PR DESCRIPTION
**Pull request by phunkyfish**
## Description

Fixup for #16677 for review after it was merged.

No functional change

## How Has This Been Tested?

OSX, latest Alpha.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/pvr/PVRManager.h [hunk 1]
```diff
@@ -556,27 +556,27 @@ namespace PVR
...
-    std::unique_ptr<CPVRGUIInfo>   m_guiInfo;                     /*!< pointer to the guiinfo data */
-    std::shared_ptr<CPVRGUIActions>              m_guiActions;                  /*!< pointer to the pvr gui actions */
-    CPVREpgContainer               m_epgContainer;                /*!< the epg container */
+    std::shared_ptr<CPVRChannelGroupsContainer> m_channelGroups;  /*!< pointer to the channel groups container */
+    std::shared_ptr<CPVRRecordings> m_recordings;                 /*!< pointer to the recordings container */
+    std::shared_ptr<CPVRTimers> m_timers;                         /*!< pointer to the timers container */
+    std::shared_ptr<CPVRClients> m_addons;                        /*!< pointer to the pvr addon container */
+    std::unique_ptr<CPVRGUIInfo> m_guiInfo;                       /*!< pointer to the guiinfo data */
+    std::shared_ptr<CPVRGUIActions> m_guiActions;                 /*!< pointer to the pvr gui actions */
+    CPVREpgContainer m_epgContainer;                              /*!< the epg container */
```
Comment (Rechi) [inline xbmc/pvr/PVRManager.h]: comments are still vertical aligned

File: xbmc/pvr/PVRManager.h [hunk 2]
```diff
@@ -556,27 +556,27 @@ namespace PVR
...
+    CPVRManagerJobQueue m_pendingUpdates; /*!< vector of pending pvr updates */
 
-    std::shared_ptr<CPVRDatabase>                 m_database;                    /*!< the database for all PVR related data */
-    mutable CCriticalSection        m_critSection;                 /*!< critical section for all changes to this class, except for changes to triggers */
-    bool                            m_bFirstStart = true;          /*!< true when the PVR manager was started first, false otherwise */
-    bool                            m_bEpgsCreated = false;        /*!< true if epg data for channels has been created */
+    std::shared_ptr<CPVRDatabase> m_database; /*!< the database for all PVR related data */
+    mutable CCriticalSection m_critSection;   /*!< critical section for all changes to this class, except for changes to triggers */
+    bool m_bFirstStart = true;                /*!< true when the PVR manager was started first, false otherwise */
+    bool m_bEpgsCreated = false;              /*!< true if epg data for channels has been created */
```
Comment (Rechi) [inline xbmc/pvr/PVRManager.h]: comments are still vertical aligned

File: xbmc/pvr/channels/PVRChannelGroupsContainer.h [hunk 3]
```diff
@@ -79,13 +79,13 @@ namespace PVR
      * @brief Get the group containing all TV channels.
      * @return The group containing all TV channels.
      */
-    std::shared_ptr<CPVRChannelGroup> GetGroupAllTV(void)  const{ return GetGroupAll(false); }
+    std::shared_ptr<CPVRChannelGroup> GetGroupAllTV(void) const{ return GetGroupAll(false); }
```
Comment (Rechi) [inline xbmc/pvr/channels/PVRChannelGroupsContainer.h]: space is missing after `const`

File: xbmc/pvr/channels/PVRChannelGroupsContainer.h [hunk 4]
```diff
@@ -79,13 +79,13 @@ namespace PVR
...
 
     /*!
      * @brief Get the group containing all radio channels.
      * @return The group containing all radio channels.
      */
-    std::shared_ptr<CPVRChannelGroup> GetGroupAllRadio(void)  const{ return GetGroupAll(true); }
+    std::shared_ptr<CPVRChannelGroup> GetGroupAllRadio(void) const{ return GetGroupAll(true); }
```
Comment (Rechi) [inline xbmc/pvr/channels/PVRChannelGroupsContainer.h]: space is missing after `const`

File: xbmc/pvr/epg/Epg.h [hunk 5]
```diff
@@ -303,19 +303,19 @@ namespace PVR
...
+    bool m_bTagsChanged = false;        /*!< true when any tags are changed and not persisted, false otherwise */
+    bool m_bLoaded = false;             /*!< true when the initial entries have been loaded */
+    bool m_bUpdatePending = false;      /*!< true if manual update is pending */
+    int m_iEpgID = 0;                   /*!< the database ID of this table */
+    std::string m_strName;              /*!< the name of this table */
+    std::string m_strScraperName;       /*!< the name of the scraper to use */
+    mutable CDateTime m_nowActiveStart; /*!< the start time of the tag that is currently active */
+    CDateTime m_lastScanTime;           /*!< the last time the EPG has been updated */
+    mutable CCriticalSection m_critSection; /*!< critical section for changes in this table */
+    bool m_bUpdateLastScanTime = false;
```
Comment (Rechi) [inline xbmc/pvr/epg/Epg.h]: comments are still vertical aligned

**Review by Rechi — COMMENTED** [review]
I wouldn't add those changes now, as it just spams the commit history.

**Comment by ksooo** [comment]
I suggest to fix the "comments are still vertical aligned" issue in a follow-up PR - for the whole PVR component.

**Comment by phunkyfish** [comment]
Fair, I’ll make the other changes later today.

**Comment by phunkyfish** [comment]
Ok, should be ready.

**Review by ksooo — APPROVED** [review]

**Pull request merged**